pnpm-lock.yaml

```diff
@@ -2042,6 +2039,10 @@ packages:
     resolution: {integrity: sha512-RwNA9Z/7PrK06rYLIzFMlaF+l73iwpzsqRIFgbMLbTcLD6cOao82TaWefPXQvB2fOC4AjuYSEndS7N/mTCbkdQ==}
     engines: {node: '>= 0.8'}
 
+  statuses@2.0.2:
```

Comment:
can't be deduplicated yet
2.0.1 is fixed in `http-errors`, but the is a PR fixing it:
https://github.com/jshttp/http-errors/pull/126